tasks/install.yum.yml

```diff
@@ -1,3 +1,15 @@
 ---
 
-- fail: msg="RedHat is not supported now"
+- name: Install requirements (Redhat)
+  package: name={{item}}
+  with_items: [udev, openssl, zip]
+
+- name: Ensure OpenVPN is installed (RedHat).
+  package:
+    name: "openvpn"
```

Comment:
OpenVPN package on rhel or centos is only in EPEL repo. You can add to requirements `epel-release`.